cli/src/pcluster/aws/cfn.py

```diff
@@ -27,14 +27,17 @@ def __init__(self):
         super().__init__("cloudformation")
 
     @AWSExceptionHandler.handle_client_exception
-    def create_stack(self, stack_name: str, disable_rollback: bool, tags: list, template_body: str):
+    def create_stack(
+        self, stack_name: str, disable_rollback: bool, tags: list, template_body: str, parameters: list = ()
```

Comment:
why not `parameters: list = None`?

Reply:
`parameters: list = None` could work too. It was unclear if Boto3 allows a None Type `parameters` argument.
I'll try using `parameters: list = None` and see how it behaves.